docs/platform/howto/cleanup-powered-off-services.rst

```diff
@@ -0,0 +1,22 @@
+Periodic cleanup of powered-off services
+========================================
+
+Aiven platform provides a way to power-off services stop accumulating usage costs when not in use. However leaving them powered-off for an extended period of time comes with disadvantages too.
```

Comment:
1. The Aiven platform?
2. "to" missing? to power-off services TO stop accumulating
3. However, leaving them powered-off for an extended period of time comes with disadvantages.